plugins/modules/ipasudorule.py

```diff
@@ -256,8 +295,16 @@ def main():
     hostgroup = module_params_get(ansible_module, "hostgroup")
     user = module_params_get(ansible_module, "user")
     group = module_params_get(ansible_module, "group")
-    cmd = module_params_get(ansible_module, 'cmd')
-    cmdgroup = module_params_get(ansible_module, 'cmdgroup')
+    allow_sudocmds = module_params_get(ansible_module, 'allow_sudocmds')
```

Comment:
*_sudocmds and *_sudocomdgroups should all be singular. Also later on.

Reply:
Fixed